Library/Formula/bib-tool.rb

```diff
@@ -1,8 +1,8 @@
 class BibTool < Formula
   desc "Manipulates BibTeX databases"
   homepage "http://www.gerd-neugebauer.de/software/TeX/BibTool/index.en.html"
-  url "https://github.com/ge-ne/bibtool/releases/download/BibTool_2_61/BibTool-2.61.tar.gz"
-  sha256 "8eaf351f1685078345a4446346559698fb58d8d1dfdf057418e5221132f9a8a4"
+  url "https://github.com/ge-ne/bibtool/archive/BibTool_2_63.tar.gz"
```

Comment:
@DomT4 That's a good question. I'm honestly not sure, but the latest version at the homepage is listed as 2.63.